resource/global/YoStarKR/resource/tasks.json

```diff
@@ -51,24 +70,6 @@
             "진입하기"
         ]
     },
-    "LE-Open": {
-        "text": [
-            "링거링에코스",
-            "스테이지",
-            "개방중"
-        ]
-    },
-    "LEChapterToLE": {
-        "ocrReplace": [
-            [
-                ".*?전당",
-                "붉은노을전당"
-            ]
-        ],
-        "text": [
-            "붉은노을전당"
-        ]
-    },
```

Comment:
This should not be removed (we are not in MaaRelease)

Reply:
Oh, I see. I've restored it again.
I'm not sure, so I'm asking a question.
Do I need to update both MaaRelease and MaaAssistantArknights for Stage Navigation?